src/app/features/creature/creature-template-spell/creature-template-spell.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@angular/core';
+import { ToastrService } from 'ngx-toastr';
+
+import { CreatureHandlerService } from '../creature-handler.service';
+import { MysqlQueryService } from '@keira-shared/services/mysql-query.service';
+import {
+  CREATURE_TEMPLATE_SPELL_ID,
+  CREATURE_TEMPLATE_SPELL_TABLE, CREATURE_TEMPLATE_SPELL_ID_2,
+  CreatureTemplateSpell
+} from '@keira-types/creature-template-spell.type';
+import { MultiRowEditorService } from '@keira-shared/abstract/service/editors/multi-row-editor.service';
+
+@Injectable()
+export class CreatureTemplateSpellService extends MultiRowEditorService<CreatureTemplateSpell> {
+
+  FIRST_ROW_START_VALUE = 0;
```

Comment:
not needed, the default is already `0` in `MultiRowEditorService`. Only specify when different ;)